src/include/daos_obj.h

```diff
@@ -1136,6 +1136,89 @@ int
 daos_oit_list(daos_handle_t oh, daos_obj_id_t *oids, uint32_t *oids_nr,
 	      daos_anchor_t *anchor, daos_event_t *ev);
 
+#define DAOS_OIT_MARKER_MAX_LEN	(32)
+
+/**
+ * Mark an object ID in the Object Index Table (OIT).
```

Comment:
Is it client side operation or server side? or both?

Reply:
the OIT table is stored at server-side, so will send RPC to server side.
currently the OIT table was implemented that each OID is one akey (with short single value update), so the daos_oit_mark() will be implemented as DAOS_COND_AKEY_INSERT update to the akey of OID.